contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -664,33 +670,33 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     /**
      * @dev Marks the receipt of a message from the given destination chain ID with the given message ID.
      *
-     * It is possible that the receipt was already received for this message, in which case we return early.
-     * If existing message is found and not yet delivered, we delete it from state and increment the fee/reward
+     * If the receipt was already previously received for this message, returns early.
+     * If existing message is found and not yet delivered, deletes it from state and increment the fee/reward
```

Comment:
```suggestion
     * If existing message is found and not yet delivered, delete it from state and increment the fee/reward
```

Reply:
I think "deletes" is correct in this case. "If the existing message is found and not yet delivered, this function deletes it from state" (consistent with "returns early" above). I'm updating the wording otherwise thought because reading it again now I think it's confusing as is.